unicode-org/cldr
# CLDR-11585 maven: refactor cldr-apps source

- completely refactor cldr-apps source locations to match the
  Maven standard layout
- retain ant as buildable (for now!)

Eclipse:
- update .classpath so that source and build work fine.
- Just ignore errors in TestShim if not on maven.

Tests:
- retains TestShim, with some comments helping with the eclipse warnings
- fix TestAnnotations to support the new source location
- update comments and fix order of assert args
- make sure web app fails properly on failure
